--- autoconf-2.50/acgeneral.m4.fhs	Tue May 22 15:42:14 2001
+++ autoconf-2.50/acgeneral.m4	Tue May 22 15:42:34 2001
@@ -829,16 +829,16 @@
 # Use braces instead of parens because sh, perl, etc. also accept them.
 AC_SUBST([bindir],         ['${exec_prefix}/bin'])dnl
 AC_SUBST([sbindir],        ['${exec_prefix}/sbin'])dnl
-AC_SUBST([libexecdir],     ['${exec_prefix}/libexec'])dnl
+AC_SUBST([libexecdir],     ['${exec_prefix}/lib'])dnl
 AC_SUBST([datadir],        ['${prefix}/share'])dnl
 AC_SUBST([sysconfdir],     ['${prefix}/etc'])dnl
 AC_SUBST([sharedstatedir], ['${prefix}/com'])dnl
-AC_SUBST([localstatedir],  ['${prefix}/var'])dnl
+AC_SUBST([localstatedir],  ['/var/lib'])dnl
 AC_SUBST([libdir],         ['${exec_prefix}/lib'])dnl
 AC_SUBST([includedir],     ['${prefix}/include'])dnl
 AC_SUBST([oldincludedir],  ['/usr/include'])dnl
-AC_SUBST([infodir],        ['${prefix}/info'])dnl
-AC_SUBST([mandir],         ['${prefix}/man'])dnl
+AC_SUBST([infodir],        ['${prefix}/share/info'])dnl
+AC_SUBST([mandir],         ['${prefix}/share/man'])dnl
 
 # Identity of this package.
 AC_SUBST([PACKAGE_NAME],
