--- autoconf-2.60/lib/autoconf/libs.m4.orig	2006-05-29 19:46:05.000000000 +0200
+++ autoconf-2.60/lib/autoconf/libs.m4	2006-06-27 23:08:13.328227000 +0200
@@ -288,7 +288,7 @@
   # See if we find them without any special options.
   # Don't add to $LIBS permanently.
   ac_save_LIBS=$LIBS
-  LIBS="-lXt $LIBS"
+  LIBS="-lX11 $LIBS"
   AC_LINK_IFELSE([AC_LANG_PROGRAM([@%:@include <X11/Xlib.h>],
 				  [XrmInitialize ()])],
 		 [LIBS=$ac_save_LIBS
@@ -299,7 +299,7 @@
 do
   # Don't even attempt the hair of trying to link an X program!
   for ac_extension in a so sl; do
-    if test -r "$ac_dir/libXt.$ac_extension"; then
+    if test -r "$ac_dir/libX11.$ac_extension"; then
       ac_x_libraries=$ac_dir
       break 2
     fi
