diff -Nru autoconf-2.58.orig/configure autoconf-2.58/configure
--- autoconf-2.58.orig/configure	2003-11-04 09:47:21.000000000 +0100
+++ autoconf-2.58/configure	2003-11-05 10:11:37.318275312 +0100
@@ -23,7 +23,7 @@
 fi
 
 # Support unset when possible.
-if ((MAIL=60; unset MAIL) || exit) >/dev/null 2>&1; then
+if ( (MAIL=60; unset MAIL) || exit) >/dev/null 2>&1; then
   as_unset=unset
 else
   as_unset=false
@@ -2180,7 +2180,7 @@
 fi
 
 # Support unset when possible.
-if ((MAIL=60; unset MAIL) || exit) >/dev/null 2>&1; then
+if ( (MAIL=60; unset MAIL) || exit) >/dev/null 2>&1; then
   as_unset=unset
 else
   as_unset=false
diff -Nru autoconf-2.58.orig/tests/wrapper.in autoconf-2.58/tests/wrapper.in
--- autoconf-2.58.orig/tests/wrapper.in	2003-09-25 11:25:02.000000000 +0200
+++ autoconf-2.58/tests/wrapper.in	2003-11-05 10:11:53.217858208 +0100
@@ -15,7 +15,7 @@
 fi
 
 # Support unset when possible.
-if ((MAIL=60; unset MAIL) || exit) >/dev/null 2>&1; then
+if ( (MAIL=60; unset MAIL) || exit) >/dev/null 2>&1; then
   as_unset=unset
 else
   as_unset=false
--- autoconf-2.58/lib/autoconf/general.m4.orig	Thu Nov 13 14:02:49 2003
+++ autoconf-2.58/lib/autoconf/general.m4	Thu Nov 13 14:03:30 2003
@@ -2157,8 +2157,7 @@
 [m4_ifvaln([$1], [AC_LANG_CONFTEST([$1])])dnl
 rm -f conftest.$ac_objext
 AS_IF([_AC_EVAL_STDERR($ac_compile) &&
-	 AC_TRY_COMMAND([test -z "$ac_[]_AC_LANG_ABBREV[]_werror_flag"
-			 || test ! -s conftest.err]) &&
+	 AC_TRY_COMMAND([test -z "$ac_[]_AC_LANG_ABBREV[]_werror_flag" || test ! -s conftest.err]) &&
 	 AC_TRY_COMMAND([test -s conftest.$ac_objext])],
       [$2],
       [_AC_MSG_LOG_CONFTEST
@@ -2198,8 +2197,7 @@
 [m4_ifvaln([$1], [AC_LANG_CONFTEST([$1])])dnl
 rm -f conftest.$ac_objext conftest$ac_exeext
 AS_IF([_AC_EVAL_STDERR($ac_link) &&
-	 AC_TRY_COMMAND([test -z "$ac_[]_AC_LANG_ABBREV[]_werror_flag"
-			 || test ! -s conftest.err]) &&
+	 AC_TRY_COMMAND([test -z "$ac_[]_AC_LANG_ABBREV[]_werror_flag" || test ! -s conftest.err]) &&
 	 AC_TRY_COMMAND([test -s conftest$ac_exeext])],
       [$2],
       [_AC_MSG_LOG_CONFTEST
